doc/stdlib/numbers.rst

```diff
@@ -150,7 +150,7 @@ Data Formats
 
    .. Docstring generated from Julia source
 
-   Convert an array of bytes to its hexadecimal representation. All characters are in lower-case. Returns a ``String``\ .
+   Convert an array of bytes to its hexadecimal representation. All characters are in lower-case. Returns an ``String``\ .
```

Comment:
Weird, I don't think I changed this, but somehow must have pulled in an old version on the rebase?